spec/paths/reports@subscription-cancellation.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  tags:
+    - Reports
+  summary: Retrieve transactions with their plan data
+  description: Retrieve transactions with their plan data
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start date
+      type: string
+      required: true
+    - name: periodEnd
+      in: query
+      description: Report end date
+      type: string
+      required: true
+    - name: aggregationField
+      in: query
+      description: Report aggregation field
+      required: true
+      type: string
+      enum:
+        - canceled-by
+        - cancel-category
+        - leads.affiliate
```

Comment:
@2e3s 
We have attribute names in camel case and enum values dash separated:

https://github.com/Rebilly/RebillyAPI/blob/master/spec/definitions/Payment.yaml#L64

There are some exceptions when it is brand name like `Visa` or `MasterCard` ro gateway name (it's a brand name too). But we always make attribute names in camel case and enum values dash separated.